### DISCUSSION TITLE
Message reply

### DISCUSSION THREAD

**Discussion by Jexly0x** [telegraf/telegraf#1625] [Q&A] [answered]
Hello. I have a simple telegram bot. There is a group to which I write messages, and the bot takes these messages from the channel and copies them into itself. If I reply to these messages in the channel, the bot sends just a message in itself, without a response. How can I implement it?

![photo](https://user-images.githubusercontent.com/97974591/169719203-1b2ed4f3-859f-4cd5-ab36-3ea76b58e42b.png)

**Comment by MKRhere** [comment, marked as answer]
This is similar to the answer to https://github.com/telegraf/telegraf/discussions/1626. You'll need a `channelMessage -> chatMessage` mapping in your DB somehow, and when a message replies to a channelMessage, you'll know which chatMessage to reply to.

**Reply by Jexly0x** [reply]
Do I need to somehow tell the bot to take that message and reply to it? (how to specify message i know reply as message?)

**Reply by MKRhere** [reply]
```TS
bot.telegram.sendMessage(chatId, message, {  reply_to_message_id: oldChatMsg.id })
```
